### PR TITLE
feat: updated joomla plugin

### PR DESCRIPTION
**Pull request by fatum**
https://app.shortcut.com/crazyegg/story/44957/joomla-plugin-needs-updating

### CODE REVIEW

File: plugin/crazyegg.xml [hunk 1]
```diff
@@ -38,4 +38,7 @@
             </fieldset>
         </fields>
     </config>
+    <updateservers>
+        <server type="extension">https://ce-development.s3.amazonaws.com/joomla/updates.xml</server>
```
Comment (shravanjoopally) [inline plugin/crazyegg.xml]: may be use crazyeggcdn.com? @imtall what do you suggest for hosting public files like these

Comment (imtall) [reply plugin/crazyegg.xml]: I think it's a good idea to use `crazyeggcdn.com` here.

@fatum - can you you set up `plugins.crazyeggcdn.com` and have that point to a new S3 bucket that contains `/joomla/updates.xml`?

Comment (fatum) [reply plugin/crazyegg.xml]: updated 👍

Comment (imtall) [reply plugin/crazyegg.xml]: Looks good to me, thanks!

File: plugin/crazyegg.xml [hunk 2]
```diff
@@ -38,4 +38,7 @@
             </fieldset>
         </fields>
     </config>
+    <updateservers>
```
Comment (shravanjoopally) [inline plugin/crazyegg.xml:41]: @fatum shoudnt this also be on plugins.crazyeggcdn.com?

Comment (fatum) [reply plugin/crazyegg.xml:41]: nice catch 👍

**Pull request merged**